tests/benchmark/test_bad_setup.py

```diff
@@ -21,32 +22,36 @@ class SomeException(Exception):
                 datasets=[iris],
                 models=[GretelLSTM],
                 config=BenchmarkConfig(
-                    working_dir="should_not_be_created",
+                    working_dir=should_not_be_created,
                 ),
             )
 
-    assert not os.path.exists("should_not_be_created")
+    assert not should_not_be_created.exists()
```

Comment:
I love your commitment to `pathlib` over `os` 😄 👍

Reply:
Haha, yeah :D
It's interesting though, with Python's dynamic typing, anytime I need to do `something.foo()` it makes me a bit nervous, as the `something` can be anything, but `os.path.exists("foo")` feels safer :)